rm_reporting/controllers/response_chasing_controller.py

```diff
@@ -31,6 +31,7 @@
     "Respondent Telephone",
     "Respondent Email",
     "Respondent Account Status",
+    "Time of Status Change",
```

Comment:
Same as the other repo, terms are getting a bit confused, we use status here but state in the table

Reply:
This has been updated